android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -149,9 +149,7 @@ function onStateChange(change) {
   keyman.refreshOskLayout();
 
   fragmentToggle = (fragmentToggle + 1) % 100;
-  if(change != 'configured') { // doesn't change the display; only initiates suggestions.
-    window.location.hash = 'refreshBannerHeight-'+fragmentToggle+'+change='+change;
-  }
```

Comment:
So 'configured' will also trigger a refreshBannerHeight now, is that okay?

Reply:
Oh, I guess I did misread it slightly.  Thought that only affected the value of `change` sent in the message for some reason.

Should be fine anyway, but... 🤷